### ISSUE TITLE
We need macOS Catalina version to run! All previous cannot be run any more!

### ISSUE THREAD

**Issue by LNAPI** [greymass/eos-voter#712]
We need macOS Catalina version to run! All previous cannot be run any more!

**Comment by aaroncox** [comment]
Hrm interesting, I hadn't upgraded yet and hadn't encountered that issue.

I'll start working on that this weekend - and we'll get a release out ASAP.

**Comment by LNAPI** [comment]
Yeah, things got pretty tough (and secure) with this new security scrutiny by new OS.
Thank you, mankind can't be left without major EOS wallet. Waiting for your relief!)

**Comment by aaroncox** [comment]
Alright, I think we've got it squared away and a new release is available here:

https://github.com/greymass/eos-voter/releases/tag/v0.7.12

If you wouldn't mind, when you get a chance see if that resolves the issue for you. 

I've upgraded to Catalina on my macOS development machine and it seems to work now for me - but some additional confirmation would be great. Thanks!

**Comment by LNAPI** [comment]
Brilliant! Fully solved! Thank you.
